bench/patches/v5/update_archived_sites.py

```diff
@@ -40,7 +40,7 @@ def execute(bench_path):
 		os.makedirs(new_directory)
 
 	for archived_site_path in old_directory.glob("*"):
-		shutil.move(archived_site_path, new_directory)
+		shutil.move(str(archived_site_path), str(new_directory))
```

Comment:
We could probably get rid of `shutil.move` altogether? 🤔 

```suggestion
		archived_site_path.rename(new_directory)
```

Ref: https://docs.python.org/3/library/pathlib.html#pathlib.Path.rename

Reply:
Yes, that could be a better solution.

Reply:
> Ref: https://docs.python.org/3/library/pathlib.html#pathlib.Path.rename

Wouldn't `replace` be a better than `rename`?
https://docs.python.org/3/library/pathlib.html#pathlib.Path.replace

Reply:
> Wouldn't `replace` be a better than `rename`?

I'm not sure how they differ. What I get from the documentation is that they are pretty much the same. If they are, I feel rename feels more natural to my mind than replace...given we may very rarely actually replace something.

Reply:
> I'm not sure how they differ. What I get from the documentation is that they are pretty much the same.

The only difference I could find from the documentation:

Rename: 
**On Unix**, if target exists and **is a file**, it will be replaced silently if the user has permission.

Replace:
If target points to an **existing file or directory**, it will be unconditionally replaced.

Suggested `replace` since we are dealing with directories.

Reply:
@rtdany10  Okay, I tried this with `.rename` and `.replace` with existing directories. Both failed the same way.

<img width="716" alt="Screenshot 2021-12-20 at 2 36 46 PM" src="https://user-images.githubusercontent.com/36654812/146741503-70eaf9bd-142c-4363-9aa2-6aa8c6bd9288.png">

Reply:
@gavindsouza I guess we can merge this PR then?